app/github/utils.py

```diff
@@ -280,10 +281,21 @@ def search(query):
 
 
 def get_issue_comments(owner, repo, issue=None, comment_id=None):
-    """Get the comments from issues on a respository."""
+    """Get the comments from issues on a respository.
+    PLEASE NOTE CURRENT LIMITATION OF 100 COMMENTS.
```

Comment:
is it work traversing comments as part of the MVP? 

i dont think a lot of our open issues have > 100 comments.. but its still good to have the piece of mind of knowing that this works..